shared/js/background/events/referrer-trimming.js

```diff
@@ -49,11 +52,7 @@ module.exports = function limitReferrerData (e) {
 
     const requestHeaders = e.requestHeaders.filter(header => header.name.toLowerCase() !== 'referer')
     if (!!tab && (!tab.referrer || tab.referrer.site !== tab.site.url)) {
-        tab.referrer = {
-            site: tab.site.url,
-            referrerHost: new URL(referrer).hostname,
-            referrer: modifiedReferrer
-        }
+        tab.referrer = new Referrer(tab.site.url, new URL(referrer).host, modifiedReferrer)
```

Comment:
Did you mean to change it from `new URL(referrer).hostname` to `new URL(referrer).host`? If so, why?

Reply:
I did not, thanks for catching :)